src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -142,6 +145,19 @@ private static boolean containsDefaultCaseLabelElement(DetailAST detailAst) {
         }).isPresent();
     }
 
+    /**
+     * Checks if a switch block contains a null case label.
+     *
+     * @param detailAst first case group to check.
+     * @return true if switch block contains null case label
+     */
+    private static boolean containsNullCaseLabelElement(DetailAST detailAst) {
+        return TokenUtil.findFirstTokenByPredicate(detailAst, ast -> {
```

Comment:
```suggestion
        return TokenUtil.findFirstTokenByPredicate(detailAst, MissingSwitchDefault::hasNullCaseLabel)
                .isPresent();
```

Reply:
Passing referenced method raises a problem, I am passing ast. getFirstChild(), if I will  use this `MissingSwitchDefault::hasNullCaseLabel ` I cannot modify parameters also in the hasNullCaseLabel, there will be many conditions. We can use the current one, I am changing hasNullCaseLabelMethod() to use inline return statements.